using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -10,6 +10,8 @@ This API synchronizes the provided customer identifier type key and value with t
 {% tab title="Android" %}
 ### syncIdentifier
 
+**Tip**: Both the identifier type and identifier should have valid string values (non null or empty), otherwise this identifier is ignored by the Identity extension.
```

Comment:
I know we use `**Tip**` in other places, but I would suggest using `{% hint style="info" %}` instead to really call it out. We can do this in another PR as there are many uses of Tip, if you agree to change them.

Reply:
- "Hint" is fine with me as well. 

- Please reword so that it reads as follows:
"If the identifier type and identifier do not have valid string values (non null or empty), the identifier is ignored by the Identity extension."